### ISSUE TITLE
undefined method `prerecord' for Minitest::Reporters::DelegateReporter in 5.10

### ISSUE THREAD

**Issue by wollkind** [seattlerb/minitest#659]
We upgraded to minitest 5.10 this morning and got this error and trace in all our tests.  I've temporarily locked to 5.9.1 until I understand what's going on here.

```
/home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:667:in `block in prerecord': undefined method `prerecord' for #<Minitest::Reporters::DelegateReporter:0x000000034df000> (NoMethodError)
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:666:in `each'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:666:in `prerecord'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:322:in `run_one_method'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:310:in `block (2 levels) in run'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:309:in `each'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:309:in `block in run'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:349:in `on_signal'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:336:in `with_info_handler'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:308:in `run'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:158:in `block in __run'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:158:in `map'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:158:in `__run'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:135:in `run'
	from /home/steve/workspace-git/cg-ruby-sem/vendor/bundle/ruby/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:62:in `block in autorun'


```

**Comment by canoe-drew** [comment]
Same problem for me.
```
/var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:667:in `block in prerecord': undefined method `prerecord' for #<Minitest::Reporters::DelegateReporter:0x00000005d66ff0> (NoMethodError)
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:666:in `each'
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:666:in `prerecord'
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:322:in `run_one_method'
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:310:in `block (2 levels) in run'
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:309:in `each'
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:309:in `block in run'
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:349:in `on_signal'
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:336:in `with_info_handler'
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:308:in `run'
	from /var/lib/gems/2.3.0/gems/railties-5.0.0.1/lib/rails/test_unit/line_filtering.rb:11:in `run'
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:158:in `block in __run'
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:158:in `map'
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:158:in `__run'
	from /var/lib/gems/2.3.0/gems/minitest-5.10.0/lib/minitest.rb:135:in `run'
	from /var/lib/gems/2.3.0/gems/railties-5.0.0.1/lib/rails/commands/test.rb:9:in `<top (required)>'
	from /var/lib/gems/2.3.0/gems/activesupport-5.0.0.1/lib/active_support/dependencies.rb:293:in `require'
	from /var/lib/gems/2.3.0/gems/activesupport-5.0.0.1/lib/active_support/dependencies.rb:293:in `block in require'
	from /var/lib/gems/2.3.0/gems/activesupport-5.0.0.1/lib/active_support/dependencies.rb:259:in `load_dependency'
	from /var/lib/gems/2.3.0/gems/activesupport-5.0.0.1/lib/active_support/dependencies.rb:293:in `require'
	from /var/lib/gems/2.3.0/gems/railties-5.0.0.1/lib/rails/commands/commands_tasks.rb:138:in `require_command!'
	from /var/lib/gems/2.3.0/gems/railties-5.0.0.1/lib/rails/commands/commands_tasks.rb:95:in `test'
	from /var/lib/gems/2.3.0/gems/railties-5.0.0.1/lib/rails/commands/commands_tasks.rb:49:in `run_command!'
	from /var/lib/gems/2.3.0/gems/railties-5.0.0.1/lib/rails/commands.rb:18:in `<top (required)>'
	from /var/lib/gems/2.3.0/gems/activesupport-5.0.0.1/lib/active_support/dependencies.rb:293:in `require'
	from /var/lib/gems/2.3.0/gems/activesupport-5.0.0.1/lib/active_support/dependencies.rb:293:in `block in require'
	from /var/lib/gems/2.3.0/gems/activesupport-5.0.0.1/lib/active_support/dependencies.rb:259:in `load_dependency'
	from /var/lib/gems/2.3.0/gems/activesupport-5.0.0.1/lib/active_support/dependencies.rb:293:in `require'
	from /home/ajones/projects/idq-website/rails/bin/rails:9:in `<top (required)>'
	from /var/lib/gems/2.3.0/gems/activesupport-5.0.0.1/lib/active_support/dependencies.rb:287:in `load'
	from /var/lib/gems/2.3.0/gems/activesupport-5.0.0.1/lib/active_support/dependencies.rb:287:in `block in load'
	from /var/lib/gems/2.3.0/gems/activesupport-5.0.0.1/lib/active_support/dependencies.rb:259:in `load_dependency'
	from /var/lib/gems/2.3.0/gems/activesupport-5.0.0.1/lib/active_support/dependencies.rb:287:in `load'
	from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from -e:1:in `<main>'
```

**Comment by zenspider** [comment]
Issues https://github.com/kern/minitest-reporters/issues/219 and https://github.com/jekyll/jekyll/pull/5615 and https://github.com/TheProlog/prolog-dry_types/issues/7

**Comment by zenspider** [comment]
Just released 5.10.1 with a kludge in place to deal with this issue.

**Comment by wollkind** [comment]
Thanks!
